br/pkg/streamhelper/client.go

```diff
@@ -19,6 +24,139 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	SeverityError            = "ERROR"
```

Comment:
what do you think of replacing `Severity` to be `Reason` and `REGULAR_OPERATION` to be `MANUAL`

Reply:
LGTM, and the TiKV side will be implemented later.